src/ng/interpolate.js

```diff
@@ -340,8 +374,13 @@ function $InterpolateProvider() {
 
       function parseStringifyInterceptor(value) {
         try {
-          value = getValue(value);
-          return allOrNothing && !isDefined(value) ? value : stringify(value);
+          if (contextAllowsConcatenation && singleExpression) {
+            // No stringification in this case, to keep the trusted value until unwrapping.
+            return value;
```

Comment:
This basically makes `parseStringifyInterceptor` a noop, can we avoid `parseStringifyInterceptor` (and any interceptor) completely in that case? Maybe only if that case is common enough that it's worth avoiding interceptors...

Reply:
Difficulty is that we assign the interpolator to the parseFn before we have calculated how many items are in `contat` and `expressionPositions` so we don't know whether `singleExpression` is true until we have finished.

Reply:
But I managed it...

Reply:
If it's difficult or ugly then we can leave it out of this PR and try later since it's not really directly related to this PR